drivers/SmartThings/zigbee-switch/src/aqara/init.lua

```diff
@@ -70,9 +112,36 @@ local function is_aqara_products(opts, driver, device)
   return false
 end
 
-local function emit_power_consumption_report_event(device, value)
-  local raw_value = value.value -- 'Wh'
+local function private_mode_handler(driver, device, value, zb_rx)
```

Comment:
can you help me understand what `private mode` is intended to do?

Reply:
The device has an operating mode internally. Depending on this value, the cluster that the device reports for the event is determined. 
Let's take a smart plug's power consumption report as an example. 
- If the private mode value is 1, it is reported to the AnalogInput cluster. 
- If the value is 0, it is reported to the ElectricalMeasurement cluster.

Reply:
is there a reason I'd want this reported on one cluster or another?

Reply:
Yes. By this value, a device report power usage and power consumption, depending on which clusters are reported. This is a operation condition of the device.